src/project/manifest/mod.rs

```diff
@@ -1022,16 +1047,15 @@ impl<'de> Deserialize<'de> for ProjectManifest {
                 name: EnvironmentName::Default,
                 features: Vec::new(),
                 features_source_loc: None,
-                solve_group: None,
+                solve_group: Some(solve_groups.add("default", 0)),
             });
             environments.by_name.insert(EnvironmentName::Default, 0);
         }
 
         // Add all named environments
+        // Note: if the default Environment is redefined, but does not specify a solve-group,
+        // it is not added to the 'default' solve-group.
```

Comment:
Is this OK?

Or do we want to add it to the 'default' solve-group? That is, the following

```toml
[environments]
default = ["somefeature"]
```

Would be parsed as if the TOML contained

```toml
[environments]
default = { features = ["somefeature"], solve-group = "default" } 
```